### DISCUSSION TITLE
Work on Mina integration

### DISCUSSION THREAD

**Discussion by Alexnpm** [trustwallet/wallet-core#1844] [Q&A]
How to start working on Mina integration to the Trust Wallet app ?
I'm software developer. I would like to work on the Android project.

**Comment by hewigovens** [comment]
Please take a look at documents https://developer.trustwallet.com/wallet-core, we need to implement it in WalletCore first

**Reply by Simon5050** [reply]
MINA integrated on trustwallet is a great idea 💡

**Comment by Simon5050** [comment]
Check out this  documents  https://developer.trustwallet.com/wallet-core, we need to implement it in WalletCore first

**Comment by Anaghara20** [comment]
Mina integrated in trustwallet is an excellent idea.
Am with u trust wallet

**Reply by Simon5050** [reply]
Yes definitely
